zebra-chain/src/transaction.rs

```diff
@@ -109,7 +109,7 @@ pub enum Transaction {
         /// The sapling shielded data for this transaction, if any.
         sapling_shielded_data: Option<sapling::ShieldedData<sapling::PerSpendAnchor>>,
     },
-    /// A `version = 5` transaction, which supports `Sapling` and `Orchard`.
+    /// An Orchard transaction (`version = 5`), which also supports Sapling.
```

Comment:
Comment Fix: These are network upgrade names, not shielded pool names. (As of NU5, we've stopped naming network upgrades after shielded pools.)

```suggestion
    /// An NU5 transaction (`version = 5`), which supports Orchard and Sapling (but not Sprout).
```

Reply:
Thanks, I wasn't aware of that. I just wanted to make the comment consistent with the ones for other tx versions above.